packages/lavapack/src/runtime.js

```diff
@@ -70,6 +70,17 @@
 })({ onStatsReady }) : () => {}
   const createKernel = // LavaMoat Prelude
 (function () {
+  const globalRef = globalThis;
+
+  const {
+    includeSnow,
+  } = {"includeSnow":false}
+
+  const SNOW = globalRef.SNOW;
+  if (!SNOW && includeSnow) {
```

Comment:
`includeSnow` is always `false` here right? Would it make sense to remove the no-op snow-specifics here in `runtime.js` until there is a way to set it picks up build-time configuration like the other entrypoints?